airbyte-integrations/connectors/source-bamboo-hr/__init__.py

```diff
@@ -0,0 +1,3 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
```

Comment:
No empty line in the end.

Reply:
Just to be clear, you want me to add an empty line at the end?